examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL/SDL.h>
```

Comment:
> will fix it now

pushed to `develop`: https://github.com/ruslo/hunter/commit/964a87e3e0f1af6dade2c13a1007274c39fa37a2